README.md

```diff
@@ -87,19 +87,19 @@ nlsolve(f!, initial_x)
 ```
 
 Alternatively, you can construct an object of type
-`DifferentiableMultivariateFunction` and pass it to `nlsolve`, as in:
+`DifferentiableVector` and pass it to `nlsolve`, as in:
```

Comment:
Why this name? It's supposed to be a  function-ish object still?

Reply:
Well, in Optim these are just called `OnceDifferentiable` for example. I think the `Function` makes in unnecessarily long to write. Here in NLsolve i figured... I'm not sure what I figure, I just found the "Multivariate" terminologically wrong (it's multivalued if anything). Could just call it Differentiable? I don't know. Happy to bikeshed.